tests/scripts/thread-cert/test_mle_msg_key_seq_jump.py

```diff
@@ -90,8 +90,16 @@ def test(self):
         #-------------------------------------------------------------------
         # Form the network.
 
+        # This test validates MLE behavior when key sequence is
+        # changed, and emulates multiple key sequence jumps within a
+        # short time window. To expedite testing and avoid long test
+        # run time (waiting for key switch guard time to expire),
+        # the "key switch guard time" is explicitly set to zero on
+        # all nodes.
+
         for node in nodes:
             node.set_key_sequence_counter(0)
+            node.set_key_switch_guardtime(0)
```

Comment:
This statement and new comments should be removed, likely: key switch guard time (`mKeySwitchGuardTime`) value is not relevant - when a new node starts up, I expect

* `mKeySwitchGuardTimer` set to `0`.
* `mKeySwitchGuardTime` set to `624`.

Only the first item is used for making a tKSC+1 decision, not the 2nd. And for authoritative MLE messages as tested here, the bigger tKSC jump (e.g. +5) happens without looking at `mKeySwitchGuardTimer` as the spec describes.

Reply:
At the end of the test, we have a case going from `20` to `21`.
I changed that to go to `22` and with that we wont need to set guard time to zero .

Reply:
ok,, thanks